### PR TITLE
PD-112: getting started create UI config article

### PR DESCRIPTION
**Pull request by micjohnson777**
This PR introduces the new UI Configuration article to the GettingStarted/Install/ articles. This article covers the initial UI configuration areas, in sequence, new users should complete as part of their SCALE deployment.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/_includes/DirectoryServicesInstallRequirementsSCALE.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+* Domain name for where the Active Directory server is located
+* Authentication credentials for access to the Active Directory server (admin user name and password to allow SCALE to communicate with the server)
+* List of trusted domains SCALE should allow
+{{< /expand >}}
+{{< expand "LDAP" "v" >}}
+Provide the following information to configure SCALE LDAP access:
+
+* Host name where the LDAP sever is located and where SCALE accesses it
+* Base and bond distinguished names (DN) and the bind password which are the authentication credentials
```
Comment (clairewrtr) [inline content/_includes/DirectoryServicesInstallRequirementsSCALE.md:14]:
```suggestion
* Base and bind distinguished names (DN) and the bind password which are the authentication credentials
```

**Review by clairewrtr — COMMENTED** [review]
Looks good, one minor change.

File: content/SCALE/GettingStarted/Install/UIConfigurationSCALE.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+## Setting Up Storage
+
+TrueNAS SCALE requires at least one storage pool. We recommend creating the required pool and then planning the rest of your storage needs before you begin adding sharing, container applications, virtual machines, or storing data.
+When planning your data storage, consider the type of data sharing you want to do, any container applications you might want to deploy, and how you want to organize stored data. 
+
+The storage creation process begins with creating a pool and then adding the datasets as needed. 
+You must create at least on storage pool [explained here]]({{< relref "SetUpStorageScale.md" >}}) on your system. 
+
+After adding your first pool, the root parent dataset is automatically created. 
+Select this root parent dataset to when you create your first dataset.
```
Comment (clairewrtr) [inline content/SCALE/GettingStarted/Install/UIConfigurationSCALE.md:77]:
```suggestion
Select this root parent dataset when you create your first dataset.
```

**Comment by micjohnson777** [comment]
If this passes technical review this can be merged into the new branch for 22.12.1 release.

**Review by Mrt134 — APPROVED** [review]
I've finished some initial reviews and am ready to merge into the 22.12.1 staging branch for further build testing and cohesiveness reviews.

**Pull request merged**